Makefile

```
@@ -151,21 +151,48 @@ docker-push: ## Push docker image with the manager.
 
-uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl $(KUBECONFIG_OPTS) delete -f - || true
+ifeq ($(FREE_RUN),1)
+APPLY_CMD = tee $(FREE_RUN_DIR)/$(1).yaml > /dev/null
+DELETE_CMD = tee $(FREE_RUN_DIR)/$(1).yaml > /dev/null
+else
+APPLY_CMD = kubectl $(KUBECONFIG_OPTS) apply -f -
+DELETE_CMD = kubectl $(KUBECONFIG_OPTS) delete -f - || true
+endif
+
+install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config. Set FREE_RUN=1 to render to FREE_RUN_DIR instead of applying.
```

checkmake (0.3.2)

[warning] Target "install" should be declared PHONY.

(phonydeclared)